cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: "http://localhost:8080",
```

Comment:
```suggestion
```

You don't need this as Cypress attempts to open html on its own webserver by default